install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -14,6 +14,9 @@ citadelHealthCheck: false
 # 90*24hour = 2160h
 workloadCertTtl: 2160h
 
+# Citadel default behavior if the citadel.istio.io/managed namespace label not found
+unlabeledNamespacesManaged: true
```

Comment:
should we opt-in to Citadel or opt-out by default?
I find opt-in behavior easier to reason as I assume existing namespaces would not be labeled accordingly.

Reply:
Wouldn't opt-out match the current behavior (i.e. `manageUnlabeld=true` as the default)? As it is, Citadel will operate on all namespaces it gets updates about (i.e. those specified in `labeled-namespaces`), labeled or not.